homeassistant/components/sia/alarm_control_panel.py

```diff
@@ -108,6 +109,11 @@ def __init__(
 
         self._attr_state: StateType = None
         self._old_state: StateType = None
+        self._attr_supported_features = (
+            AlarmControlPanelEntityFeature.ARM_AWAY
+            | AlarmControlPanelEntityFeature.ARM_NIGHT
+            | AlarmControlPanelEntityFeature.ARM_CUSTOM_BYPASS
```

Comment:
This entity doesn't seem to implement the methods for this?
Calling the `alarm_control_panel.alarm_arm_away`, `alarm_control_panel.alarm_arm_night`, or `alarm_control_panel.alarm_custom_bypass` will now throw an `NotImplemented()` error.